examples/substrait-spark/app/build.gradle

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file was generated by the Gradle 'init' task.
```

Comment:
minor/non-blocking: For consistency it would be nice to use the Kotlin DSL and not the Groovy DSL, as that is what the other builds use.

Reply:
No problem with consistency;  by habit I use Groovy, but can check out the Kotlin spec can convert